argocd/infrastructure/geocoder-api.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: geocoder-api-deployment
```

Comment:
What namespace will this application be installed in?

Reply:
The `argocd`-namespace

Reply:
This seems to be best practice, although they have a guide for putting them in any other namespace:
https://argo-cd.readthedocs.io/en/stable/operator-manual/app-any-namespace/
They do warn that misconfiguration could lead to vulnerabilities though. We should therefore have a good reason for putting applications in a namespace other than `argocd` IMO.

Reply:
Aiaiai...I don't like the idea that all our applications must be in the same namespace in kubernetes. That will prevent us from separating applications and their priveleges.

See if we can work around this to be able to deploy applications/services to multiple namespaces.

Reply:
Fixed by https://github.com/openearthplatforminitiative/infrastructure/pull/5